Agents/FeatureInfoAgent/code/src/main/java/com/cmclinnovations/featureinfo/Utils.java

```diff
@@ -0,0 +1,109 @@
+package com.cmclinnovations.featureinfo;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import uk.ac.cam.cares.jps.base.timeseries.TimeSeries;
+import uk.ac.cam.cares.jps.base.timeseries.TimeSeriesClient;
+
+/**
+ * Misc utilities.
+ */
+public final class Utils {
+    
+    /**
+     * Logger for reporting info/errors.
+     */
+    private static final Logger LOGGER = LogManager.getLogger(Utils.class);
+
+    /**
+     * Constructor.
+     */
+    private Utils() {
+        // No
+    }
+
+    /**
+     * Combine timeseries objects into a single timeseries objects. For some reason this is
```

Comment:
This is specific to the river sensor stations due to the way I initialised them, one station may contain multiple tables, typically one station will contain a table with multiple columns if it measures more than 1 thing. It's a combination of laziness and how the API returns data

Reply:
If this code is run on non-river sensor stations what will happen? 

This agent is only going to be able to handle timeseries if timeseries are storred generically for all station/location types. I don't really want to be adding an ever growing list of "if(thisTypeOfStation){}" clauses.

Reply:
Based on the way you create the time series objects it won't make any difference, but it's a bit inefficient

Reply:
I'd suggest we leave it as-is then. For this PR I'm more interested in getting the functionality correct than it being more efficient.